### PR TITLE
the waterMark by file to add to the API

### PR DESCRIPTION
**Pull request by Palesa2lo**
Watermark API models.

### CODE REVIEW

File: src/SigniFlow.Connect/Model/AddWatermarkByFileRequest.cs [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        /// To validate all properties of the instance
+        /// </summary>
+        /// <param name="validationContext">Validation context</param>
+        /// <returns>Validation Result</returns>
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            yield break;
+        }
+    }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Model/AddWatermarkByFileRequest.cs:222]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Model/AddWatermarkByFileRequestTextRGBColorField.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        /// To validate all properties of the instance
+        /// </summary>
+        /// <param name="validationContext">Validation context</param>
+        /// <returns>Validation Result</returns>
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            yield break;
+        }
+    }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Model/AddWatermarkByFileRequestTextRGBColorField.cs:103]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Model/AddWatermarkByFileResponse.cs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        /// To validate all properties of the instance
+        /// </summary>
+        /// <param name="validationContext">Validation context</param>
+        /// <returns>Validation Result</returns>
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            yield break;
+        }
+    }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Model/AddWatermarkByFileResponse.cs:108]: RA3 - Spaaaaace

**Review by Riccorbypro — CHANGES_REQUESTED** [review]
Please implement the missing tests

File: src/SigniFlow.Connect/Model/AddWatermarkByFileResponse.cs (hunk 3 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Model/AddWatermarkByFileResponse.cs:108]: This is in generated code

File: src/SigniFlow.Connect/Model/AddWatermarkByFileRequestTextRGBColorField.cs (hunk 2 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Model/AddWatermarkByFileRequestTextRGBColorField.cs:103]: This is in generated code

File: src/SigniFlow.Connect/Model/AddWatermarkByFileRequest.cs (hunk 1 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Model/AddWatermarkByFileRequest.cs:222]: This is in generated code

**Review by Signi-TK — APPROVED** [review]

**Review by marko3645 — APPROVED** [review]

**Review by nataleap — APPROVED** [review]

**Review by MorneWilken — APPROVED** [review]

**Pull request merged**